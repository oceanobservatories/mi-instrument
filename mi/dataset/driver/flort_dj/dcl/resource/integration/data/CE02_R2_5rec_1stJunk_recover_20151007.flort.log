2015/10/07 18:41:15.296 [flort:DLOGP2]:Logger started: Idle state, without initialize
2015/10/07 18:45:14.482 [flort:DLOGP2]:Instrument Started [Power On]
2015/10/07 18:45:16.089 07/15	18:43:33	700	128	695	69	460	47	543
2015/10/07 18:45:17.223 10/07/15	18:43:34	700	127	695	66	460	48	543
2015/10/07 18:45:18.358 10/07/15	18:43:35	700	127	695	65	460	47	543
2015/10/07 18:45:19.492 10/07/15	18:43:36	700	133	695	67	460	48	543
2015/10/07 18:45:20.632 10/07/15	18:43:37	700	132	695	70	460	48	543
2015/10/07 18:45:21.766 10/07/15	18:43:38	700	127	695	68	460	49	542
2015/10/07 18:48:03.136 [flort:DLOGP2]:Instrument Stopped [Power Off]
