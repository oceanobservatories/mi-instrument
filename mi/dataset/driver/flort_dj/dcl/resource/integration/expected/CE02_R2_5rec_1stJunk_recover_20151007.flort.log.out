
 subsite  | node  | sensor       | bin        | method    | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-----------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE02SHSM | RID27 | 02-FLORTD000 | 3653164800 | recovered | 3653232317.2230000496 |          2 |         3653232214 | 3653232317.2230000496 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               543 |             127 |              48 |             66 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3653164800 | recovered | 3653232318.3579998016 |          2 |         3653232215 | 3653232318.3579998016 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               543 |             127 |              47 |             65 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3653164800 | recovered |  3653232319.492000103 |          2 |         3653232216 |  3653232319.492000103 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               543 |             133 |              48 |             67 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3653164800 | recovered | 3653232320.6319999695 |          2 |         3653232217 | 3653232320.6319999695 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               543 |             132 |              48 |             70 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3653164800 | recovered |  3653232321.765999794 |          2 |         3653232218 |  3653232321.765999794 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               542 |             127 |              49 |             68 |        null

(5 rows)
